WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -558,25 +558,10 @@ - (BOOL)wasOnboardingShown
 
 - (void)showOnboardingTips
 {
-    AMPopTip *popTip = [AMPopTip popTip];
     CGFloat xValue = IS_IPAD ? CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-20.0 : CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-10.0;
```

Comment:
Know this code is from before, but I'd consider breaking this up as suggested above as well.